src/daemon/bitcoind/poller.rs

```diff
@@ -1019,6 +1028,7 @@ fn handle_spent_unvault(
             }
         }
         None => {
+            // We don't remove it from the cache, so we'll check this outpoint at the next poll
```

Comment:
Not sure but iirc there is another call site where this has to be noted?

Could you make this "normal behaviour": ie not logging as error, and saying something like:
> Could not find a current transaction spending the Unvault txo at '{}', will check again at next poll

Reply:
> Not sure but iirc there is another call site where this has to be noted?

`unvault_spender` is called only here, `git grep unvault_spender`:
```
src/daemon/bitcoind/poller.rs:fn unvault_spender(
src/daemon/bitcoind/poller.rs:    match unvault_spender(revaultd, bitcoind, previous_tip, &unvault_outpoint)? {

```